trunk/SUAVE/Methods/Weights/Correlations/FLOPS/wing.py

```diff
@@ -99,7 +100,7 @@ def wing_weight_FLOPS(vehicle, wing, WPOD, complexity, settings, num_main_wings)
     else:
         NSD             = 500
         N2              = int(sum(networks.wing_mounted) / 2)
-        ETA, C, T, SWP  = generate_wing_stations(vehicle.fuselages['fuselage'].width, wing)
+        ETA, C, T, SWP  = generate_wing_stations(vehicle.fuselages['fuselage'].width, copy.deepcopy(wing))
```

Comment:
Not the most elegant solution, but generate_wing_stations adds segment information to the wing. I don't think FLOPS should be changing anything about the geometry as defined by the user. By passing a copy of the wing, the original geometry does not change and all the calculations still take place properly.